.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+name: IOC build and release
+permissions:
+  contents: write
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build
+        run: docker compose up
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: rffe-epics-ioc-*.tar.gz
```

Comment:
Have you been able to test if this works as we expect?

Reply:
No. I can try and add it to my test repository to see what happens